### ISSUE TITLE
Unable to find price for Flipkart items

### ISSUE THREAD

**Issue by dylants** [dylants/price-finder#98]
The Flipkart site no longer works, tests fail.

**Comment by dylants** [comment]
After looking into this a bit, it looks as though Flipkart has moved away from rendering the page content we need on initial render, and instead performs these operations via an async javascript API call.  You can see this by loading any product.  You'll notice it takes a moment for the price to appear.

That can be okay, if the API call it makes is open to anyone.  But it appears the API call it uses is a `POST` request (I know right, why a `POST`?) to the following URL: `https://www.flipkart.com/api/3/page/dynamic/product`.  In testing this locally, the endpoint returns a 403 (Forbidden), so it appears it also requires some authentication via a cookie (or header, or something).

So, I feel as though the site is _really_ trying to not allow anyone to access their pricing data.

In a last ditch effort to provide some detail, I noticed that the `meta` tag for `Description` contains a string which includes the price.  If we apply regex, we can get the price here, though the normal regex problems apply (this is very fragile, will most likely break soon, etc, etc).  But it is a way to get the price... sooo....

In other news, getting the name or category is pretty much a fail at this point.  I'm going to just grab the title for the name and call it a day.

TL;DR:  We've almost no way to scrape this site, but there is one way that sucks.  So I guess we'll use that.
